src/components/AddView.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-const AddView = () => {
+const AddView = ({ token }) => {
```

Comment:
It doesn't seem that the token is being used in this component? Therefore, there is no need to import it I would say.

Reply:
The token prop has been added in each view only because we knew the other team would need to use it. Will delete it now.

Reply:
Hi @YuriDevAT! Removing the token prop means the app will break when we merge- the other group's work is already set up to pass the prop. Is there something we can do about this?

Reply:
Since the other group already have these props in their code, the code should not break. There is no need for both of you to have the same code, because when merging you have the possibility to choose the part of code with the prop in it (merge conflict doesn't necessarily mean that something is wrong, but you have to choose with which code you are going to work). It should be possible to review branches independent from each other. It is not a mistake, of course, such a suggestion to not have parts which are not used!

Reply:
Alright, thank you.